cmd/load-generator/latency-charter.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+
+import matplotlib
```

Comment:
Is this in the stdlib or do we need to modify a `requirements` file to have it available for folks?